tests/std/tests/GH_000935_complex_numerical_accuracy/env.lst

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft Corporation.

```

Comment:
Is there any chance this is going to get used for any other floating point stuff? Might be worth extracting that part of it and moving it into another `env.lst` file which would get included.